### PR TITLE
Tighten up spatial coherence wording

### LINKED ISSUE #43: Tweak spec wording about spatial coherence
https://groups.google.com/forum/#!topic/cesium-dev/tCCooBxpZFU

From @kring:

> I noticed that the spec says, "the bounding volume for child tiles are completely inside the parent's bounding volume."  Why?  Isn't it sufficient for the child _content_ to be inside the parent's bounding volume?

### PR DESCRIPTION
**Pull request by pjcozzi**
Fixes #43.  See the new "Bounding volume spatial coherence" section.

@lilleyse can you review and merge?  This is a small change; most of it is new TOC links.

Based on discussion with @kring - https://groups.google.com/forum/#!profile/cesium-dev/APn2wQdDf8J2kNnhMkqr9zuD_pygdVUnsj_Ry_fyJdzzNnQ1nVxZA0EUyytOfBuqM3VR6JSBsHGH/cesium-dev/tCCooBxpZFU/y3nKJT9eBgAJ

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -166,14 +163,26 @@ See [schema](schema) for the detailed JSON schema for tiles.json.
...
+  Bounding spheres for the four child tiles.  The children's content are completely inside the parent's bounding volume, but the children's bounding volumes are not since they are not tightly fit.
+</p>
+
 ### Creating spatial data structures
 
 The tree defined in tiles.json by `root` and, recursively, its `children`, can define different types of spatial data structures.  In addition, any combination of tile formats and refinement approach (replacement or additive) can be used, enabling a lot of flexibility to support heterogeneous datasets.
 
-It is up to the conversion tool that generates tiles.json to define an optimal tree for the dataset.  A runtime engine, such as Cesium, is generic and will render any tree defined by tiles.json.  Here's a brief descriptions of how 3D Tiles can represent various spatial data structures.
+It is up to the conversion tool that generates tiles.json to define an optimal tree for the dataset.  A runtime engine, such as Cesium, is generic and will render any tree defined by tiles.json.  Here's a brief descriptions of how 3D Tiles can represent various spatial data structures.  This is allowed, but is suboptimal.
```
Comment (lilleyse) [inline README.md]: > This is allowed, but is suboptimal

I'm not sure what this is referring to.

Comment (pjcozzi) [inline README.md]: The giant bounding spheres.  I can just remove it if you want.  We don't need to over tutorialize this.

Comment (lilleyse) [inline README.md]: Doesn't matter to me. It just seemed in the wrong place.

Comment (pjcozzi) [inline README.md]: Removed.

**Comment by lilleyse** [comment]
The rest looks good to me, is it ready to merge?

**Comment by pjcozzi** [comment]
Yes.

**Pull request merged**